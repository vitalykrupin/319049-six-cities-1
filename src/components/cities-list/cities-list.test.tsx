import * as React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from './cities-list';

const citiesMock = [
  `Paris`,
  `Brussels`,
  `Amsterdam`,
  `Cologne`
];

it(`Cities List correctly renders`, () => {
  const tree = renderer
    .create(<CitiesList
      cities={citiesMock}
      city={`Paris`}
      onCityClick={jest.fn()}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
