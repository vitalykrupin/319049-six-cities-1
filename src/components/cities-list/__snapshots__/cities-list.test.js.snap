// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cities List correctly renders 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      onClick={[Function]}
    >
      <span>
        Paris
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      onClick={[Function]}
    >
      <span>
        Brussels
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      onClick={[Function]}
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      onClick={[Function]}
    >
      <span>
        Cologne
      </span>
    </a>
  </li>
</ul>
`;
